compiler/rustc_const_eval/src/errors.rs

```diff
@@ -87,3 +87,27 @@ pub(crate) struct TransientMutBorrowErrRaw {
     pub span: Span,
     pub kind: ConstContext,
 }
+
+#[derive(SessionDiagnostic)]
+#[error(const_eval::const_evaL_max_num_nodes_exceeded)]
+pub(crate) struct MaxNumNodesExceeded {
+    #[primary_span]
+    pub span: Span,
+    pub s: String,
```

Comment:
Can you give this a better name

Reply:
Sure thing, I'll switch it up with the next batch of fixes. How does `MaxNumNodesInConstErr` sound?

Reply:
I think it was the field that @compiler-errors meant, not the type.

Reply:
Ops, that's right. Addressed in latest commit.